src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/Maintainability/DoNotIgnoreMethodResults.cs

```diff
@@ -116,6 +117,15 @@ public sealed class DoNotIgnoreMethodResultsAnalyzer : DiagnosticAnalyzer
                                                                              isPortedFxCopRule: true,
                                                                              isDataflowRule: false);
 
+        internal static DiagnosticDescriptor LinqMethodRule = DiagnosticDescriptorHelper.Create(RuleId,
+                                                                             s_localizableTitle,
+                                                                             s_localizableMessageLinqMethod,
+                                                                             DiagnosticCategory.Performance,
+                                                                             RuleLevel.IdeSuggestion,
+                                                                             description: s_localizableDescription,
+                                                                             isPortedFxCopRule: false,
```

Comment:
I have set `false` as this is a new behavior but the rule ID is reused so maybe we should stick to `true`.

Reply:
I'd still stick with `true` here and retain consistency for the flag for a given rule ID, as this flag is used to generate a custom tag, which is then used in some post-build tools to generate documentation and such.